app/__init__.py

```diff
@@ -70,6 +81,25 @@ def add_generic_cors_header(response):
     return response
 
 
+@app.after_request
+def log_response(response):
+    logger.info(
+        "%s %s - %s",
+        request.method,
+        request.path,
+        response.status,
+        extra={
+            'response': {
+                "status_code": response.status_code,
+                "headers": dict(response.headers.items()),
+                "json": response.json
+            },
+            "duration": time.time() - g.get('request_started', time.time())
+        }
+    )
+    return response
+
+
```

Comment:
This should be placed after all `@app.after_request` in order to have everythink logged. Otherwise we are missing infos in the logs, for exemple the cache control header.